sass/includes/_card-grid.scss

```diff
@@ -1,15 +1,29 @@
 .card-grid {
     @extend .row;
     display: grid;
-    grid-gap: 1rem;
-    margin: 1rem 0;
```

Comment:
Note: I've generalised this, I assumed more consistency with card-grids but I've moved them to their sepecifc modfiers.